File: C#/Client Server Plugin/MyPlugin/Forms/FormMessage.cs
using Imminent.IServerVars;
using MyPlugin.Main;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace MyPlugin.Forms
{
    public partial class FormMessage : Form
    {
        private IServerClient IServerClient;

        public FormMessage(IServerClient _IServerClient)
        {
	        InitializeComponent();
	        IServerClient = _IServerClient;
	        this.Text = string.Format("Message - {0}", IServerClient.IServerState.UserName);
        }

        private void Button1_Click(object sender, EventArgs e)
        {
            Variables.iServerNetworkActions.SendToClient(IServerClient, Main.Variables.Command.Message, TextBox1.Text);
        }
    }
}

File: VB/Client Server Plugin/MyPlugin/Forms/FormMessage.vb
Imports Imminent
Imports Imminent.GUI
Imports Imminent.IServerVars
Public Class FormMessage

    Private IServerClient As IServerClient
    Sub New(_IServerClient As IServerClient)
        InitializeComponent()
        IServerClient = _IServerClient
        Me.Text = String.Format("Message - {0}", IServerClient.IServerState.UserName)
    End Sub

    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        IServerNetworkActions.SendToClient(IServerClient, Command.Message, TextBox1.Text)
    End Sub

End Class